test/unit/simplemodel-test.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'utils/simplemodel'
+], function (simplemodel) {
+    /* jshint qunit: true */
+
+    module('simplemodel');
+
+
+    test('simplemodel', function(assert) {
+        var model = simplemodel;
```

Comment:
// Otherwise we are editing the model shared by everyone
model = _.extend({}, simpleModel);